gulpfile.js

```diff
@@ -9,7 +9,7 @@ const concat = require('gulp-concat');
 const insert = require('gulp-insert');
 const remoteSrc = require('gulp-remote-src');
 const replace = require('gulp-replace');
-const uglify = require('gulp-uglify');
+//const uglify = require('gulp-uglify');
```

Comment:
I would have expected @hound-ci to catch this but I think you need a space between `/` and `c`

```
// const
```